include/dali/c_api.h

```diff
@@ -194,6 +194,9 @@ DLL_PUBLIC void daliSetExternalInputBatchSize(daliPipelineHandle *pipe_handle, c
  * copying data onto GPU. This function is asynchronous, so it's your responsibility to
  * synchronize on a provided CUDA stream.
  *
+ * If GPU memory is provided, it is assumed to reside on the same device that was used
+ * to create the pipeline that is used. See `device_id` parameter of the `daliCreatePipeline`.
```

Comment:
```suggestion
 * If GPU memory is provided, it is assumed to reside on the same device that
 * the pipeline that is used. See `device_id` parameter of the `daliCreatePipeline`.
```
just a suggestion

Reply:
I wanted to mention the device_id that we passed to the pipeline that we are using now. Quite a mouthful, will change.

Reply:
Done